src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Color[] colors = Color.values();
+    private final Random random = new Random();
+
+    public String getRandom() {
```

Comment:
```suggestion
    public String getRandomColor() {
```
You return a color here